app/api/v1/mediator/tracker.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any
+from electionguard.group import ElementModQ
+from electionguard.serializable import read_json_object
+from electionguard.tracker import tracker_hash_to_words, tracker_words_to_hash
+from fastapi import APIRouter, Body
+
+from ..models import (
+    TrackerHash,
+    TrackerHashRequest,
+    TrackerWords,
+    TrackerWordsRequest,
+)
+from ..tags import UTILITY
+
+router = APIRouter()
+
+
+@router.post("/hash", tags=[UTILITY])
```

Comment:
Added as utility since this isn't required for any particular step. It's just a function you can use.